src/app/Fake.DotNet.Cli/DotNet.fs

```diff
@@ -487,9 +487,17 @@ module DotNet =
         }
         static member Create() = {
             DotNetCliPath =
+                let version = try Some <| getSDKVersionFromGlobalJson() with _ -> None                              
```

Comment:
Usually I don't like `with _ -> None` if some parsing error occurs I want the exception to be thrown or have a warning

Reply:
I completely agree. It's just that `getSDKVersionFromGlobalJson` fails when `global.json` is not present, which won't work here. 
I didn't want to make a bigger change until the idea was verified, but I suppose having `getSDKVersionFromGlobalJson` return `None` when there is no `global.json` and fail otherwise would make sense?